### ISSUE TITLE
update package structure

### ISSUE THREAD

**Issue by ttm** [ttm/music#14]
comply with https://packaging.python.org/en/latest/tutorials/packaging-project

**Comment by ksylvan** [comment]
@ttm Looks like you finished this issue, yes?

The link above does not work, but the simplified link now is: https://packaging.python.org/en/latest/

**Comment by ttm** [comment]
@ksylvan yes — and sorry for the long silence. Verifying it properly before closing, since "complies with the packaging tutorial" deserves evidence rather than a nod:

- `pyproject.toml` with PEP 621 metadata and `setuptools>=61` as the build backend — no `setup.py`, no `setup.cfg`
- Both artifacts built and published: [`music 1.1.1`](https://pypi.org/project/music/1.1.1/) ships a wheel and an sdist
- Wheel contains exactly `music/` and its `dist-info` — no stray top-level `tests` package, which was the actual symptom behind #15, #13, #12 and #10
- `py.typed` declared as package data, so the type hints are visible to consumers
- Trove classifiers, `requires-python = ">=3.10"`, and `project.urls` for Documentation, Tutorial, Source, Changelog and Issues
- A clean-venv `pip install music` imports and renders audio; CI runs the whole gate on 3.10 through 3.13

`tools/release.py` now enforces the parts that can drift — the version agreeing across `pyproject.toml`, `CITATION.cff` and `CHANGELOG.md`, and the tag pointing at the commit the artifacts were built from.

Closing.
